docs/advanced_tutorials/17_plotting_functionality.ipynb

```diff
@@ -97,24 +97,26 @@
     }
    ],
    "source": [
+    "from sbi.analysis.plotting_classes import HistDiagOptions, ScatterOffDiagOptions\n",
+    "\n",
     "_ = pairplot(\n",
     "    posterior_samples,\n",
     "    limits=[[-3, 3] * 3],\n",
     "    figsize=(5, 5),\n",
     "    diag=\"hist\",\n",
     "    upper=\"scatter\",\n",
-    "    diag_kwargs={\n",
-    "        \"mpl_kwargs\": {\n",
-    "            \"bins\": 10,\n",
+    "    diag_kwargs=HistDiagOptions(\n",
```

Comment:
In general in this tutorial, it would be good to add some kind of migration note, e.g., sth like “Previously you’d pass nested dicts to *_kwargs. You can still do that; but we recommend using the typed *Options for clarity and autocompletion.”